src/relayserver/RelayServer.ts

```diff
@@ -471,19 +488,19 @@ export class RelayServer extends EventEmitter {
       throw new StateError('Could not get gasPrice from node')
     }
     const balance = await this.getManagerBalance()
-    if (balance.lt(toBN(minimumRelayBalance))) {
+    if (balance.lt(toBN(defaultMinManagerBalance))) {
       throw new StateError(
         // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-        `Server's balance too low ( ${balance}, required ${minimumRelayBalance}). Waiting for funding...`)
+        `Server's balance too low ( ${balance}, required ${defaultMinManagerBalance}). Waiting for funding...`)
```

Comment:
Why did you change it to 'default'? Doesn't it break the sole idea of having it be configurable?

Reply:
Actually I didn't change it, it was never configurable and I forgot to add it to configuration. Fixed.